optical/converter/__init__.py

```diff
@@ -3,6 +3,7 @@
 license: MIT
 Created: Sunday, 28th March 2021
 """
+# TODO: needs better solution
```

Comment:
Please give a brief description of what needs a better solution.

Reply:
Changed todo with details